ckanext/canada/schemas/presets.yaml

```diff
@@ -4075,3 +4075,25 @@ presets:
     display_snippet: multiple_choice.html
     validators: scheming_multiple_choice
     output_validators: scheming_multiple_choice_output
+
+# Field = Validation Schema.
+- preset_name: canada_resource_validation_schema
+  values:
+    field_name: schema
+    form_snippet: null
+    form_panel: _exclude_from_form
+    required: false
+    display_snippet: null
+    validators: canada_blank_value_creator
+    output_validators: canada_resource_schema_output
+
+# Field = Validation Options.
+- preset_name: canada_resource_validation_options
+  values:
+    field_name: validation_options
+    form_snippet: null
+    form_panel: _exclude_from_form
+    required: false
+    display_snippet: null
+    validators: canada_blank_value_creator
```

Comment:
We need a blank value stored and can't use `ignore` because of ckanext-validation?

Reply:
Okay I got `ignore` working as the validator. I am just going to keep the `None` value output validator for existing Resources that may have values in these fields.